CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ This changelog documents all notable user-facing changes of VAST.
 
 ## Unreleased
 
+- ⚠️ The default batch size has been changed from 1000 to 1024.
```

Comment:
Can you be more specific what you mean by batch? From a user perspective, it can mean a lot things across the data processing pipeline.